src/PlusDataCollection/Andor/vtkPlusAndorVideoSource.cxx

```diff
@@ -993,6 +993,32 @@ bool vtkPlusAndorVideoSource::GetUseCosmicRayCorrection()
   return this->UseCosmicRayCorrection;
 }
 
+// ----------------------------------------------------------------------------
+PlusStatus vtkPlusAndorVideoSource::SetCameraIntrinsics(double* intrinsics)
```

Comment:
Style-wise and from type safety standpoint, it might be better to use `std::vector<double>` or `std::array<double, 9>`. Possibly also update ivar to be the same type.

Reply:
Thanks, pushed a change for this. I left the internal var as is, because I wasn't sure how to handle `Set/GetVectorAttribute` nicely in `Read/WriteConfiguration` which was complaining about expecting `double *`s.